### ISSUE TITLE
Use ExporterAddr where applicable

### ISSUE THREAD

**Issue by nicoduck** [taktv6/tflow2#23]
Hi,
please add the functionality to use the netflow field exporteraddr where set and differend from the source address.
The current implementation either tries to snmpwalk the wrong device or cannot associate incoming traffic.

Nico

**Comment by taktv6** [comment]
Good point. Will implement this as soon as I find time to get this done.
